x-pack/plugins/security_solution/public/management/pages/policy/view/policy_details.test.tsx

```diff
@@ -10,18 +10,14 @@ import { mount } from 'enzyme';
 import { PolicyDetails } from './policy_details';
 import { EndpointDocGenerator } from '../../../../../common/endpoint/generate_data';
 import { createAppRootMockRenderer } from '../../../../common/mock/endpoint';
-import { getManagementUrl } from '../../../common/routing';
+import { getPolicyDetailPath, getPoliciesPath } from '../../../common/routing';
 
 describe('Policy Details', () => {
   type FindReactWrapperResponse = ReturnType<ReturnType<typeof render>['find']>;
 
-  const policyDetailsPathUrl = getManagementUrl({
-    name: 'policyDetails',
-    policyId: '1',
-    excludePrefix: true,
-  });
-  const policyListPathUrl = getManagementUrl({ name: 'policyList', excludePrefix: true });
-  const policyListPathUrlWithPrefix = getManagementUrl({ name: 'policyList' });
+  const policyDetailsPathUrl = getPolicyDetailPath('1');
+  const policyListPathUrl = getPoliciesPath();
```

Comment:
since they are the same now, we shuld just collapse the two into one single var under `policyListPathUrl`